### PR TITLE
Add windows crossbuild in CI

### PR DESCRIPTION
**Pull request by fisherxu**
**What type of PR is this?**
/kind test

**What this PR does / why we need it**:

Now the windows release is mainly focus on the edgecore/keadm, so adding edgecore/keadm/cloudcore crossbuild check in CI, to ensure that code changes would not cause problems with compilation on windows.

In previous versions, we only compiled the windows version when we published the release.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: hack/lib/golang.sh [hunk 1]
```diff
@@ -308,6 +308,7 @@ kubeedge::golang::cross_build_place_binaries() {
         set +x
       fi
     elif [ "${goos}" == "windows" ]; then
+      sudo apt-get install -y mingw-w64
```
Comment (fisherxu) [inline hack/lib/golang.sh:311]: Copied from https://github.com/kubeedge/kubeedge/blob/master/Makefile#L446

**Comment by fisherxu** [comment]
Ready for review :) cc @WillardHu @Shelley-BaoYue @luomengY @wbc6080

**Review by Shelley-BaoYue — COMMENTED** [review]
/lgtm 
It's really helpful for checking the windows code.

**Comment by luomengY** [comment]
lgtm

**Pull request merged**